ui/app/AppLayouts/Chat/views/ChatHeaderContentView.qml

```diff
@@ -269,6 +270,8 @@ RowLayout {
                 }
             }
             asset.name: chatContentModule? chatContentModule.chatDetails.icon : ""
+            asset.isImage: chatContentModule.chatDetails.icon !== ""
+            asset.isLetterIdenticon: chatContentModule.chatDetails.icon === ""
```

Comment:
Makes sense but shouldn't these 2 booleans be "calculated" automatically inside StatusAssetSettings @alexandraB99?